auto_dev/utils.py

```diff
@@ -284,19 +284,20 @@ def remove_suffix(text: str, suffix: str) -> str:
     return text[: -len(suffix)] if suffix and text.endswith(suffix) else text
 
```

Comment:
services should be a standard at some point for commands. a lot easier to understand what is going on imo